MTM-53068 improvements and refresh in microservice sdk docs. (#1017)
SoftwareAG/c8y-docs: content/microservice-sdk/rest-bundle/microservice-development.md

}
```

> Note: there is a possibility to link multiple child assets or multiple child devices to one parent in single request. 
> Refer to the [Inventory Child Operations](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Child-operations) in the {{< openapi >}} for more details.

Reviewer: ```suggestion
{{< c8y-admon-info >}}
You can link multiple child assets or multiple child devices to the same parent in a single request. 
Refer to inventory child operations in the [Inventory API](https://{{< domain-c8y >}}/api/core/{{< c8y-current-version >}}/#tag/Child-operations) in the {{< openapi >}} for more details.
{{< /c8y-admon-info >}}
```

    }
]
```

> Note: All devices channel (*) is deprecated and should be avoided. Subscription on this channel may cause performance issues 

Reviewer 2: :+1:

> Note: All devices channel (*) is deprecated and should be avoided. Subscription on this channel may cause performance issues 

Reviewer 3: ```suggestion
Note: A channel for all devices (*) is deprecated and should be avoided. Subscription on this channel may cause performance issues 
```
I find it a bit weirdly worded. How about this?

Author: (*) means the channel ending with *, which means listening to notifications from all devices and not a single one.
Examples
/events/1331421   - channel for notifications of events from device 1331421
/events/*   - channel for notifications of events from all devices (may be too much for subscribed client to handle)

    }
]
```

{{< c8y-admon-info >}}
A channel for all devices (*) is deprecated and should be avoided. Subscription on this channel may cause performance issues

> on both the client and the server side, and this options may be removed in future versions of {{< product-c8y-iot >}}.

Reviewer 4: ```suggestion
on both the client and the server side, and this option may be removed in future versions of {{< product-c8y-iot >}}.
```

Author: done and improvement squashed with previous commit
